apps/mobile/src/icons/CirclePostIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import Svg, { Path, SvgProps } from 'react-native-svg';
```

Comment:
I think some of the screens in the figma used this old Circle icon by mistake, we should use the same icon we have in the footer (square border around the plus)